packages-ui/roosterjs-react/lib/contextMenu/menus/createImageEditMenuProvider.tsx

```diff
@@ -77,6 +78,20 @@ const ImageResizeMenuItem: ContextMenuItem<ImageEditMenuItemStringKey, ImageEdit
             }
         });
     },
+    getSelectedId: (editor, target) => {
```

Comment:
I see you are passing the node parameter for this function in the interface. Should we use the node her instead getting the selection?